snapshot/lib/snapshot/reset_simulators.rb

```diff
@@ -69,10 +60,29 @@ def self.devices
       #   iPhone 4s (FE9D6F85-1C51-4FE6-8597-FCAB5286B869) (Shutdown) (unavailable, runtime profile not found)
 
       result = all_devices.lines.map do |line|
-        (line.match(/\s+([\w\s]+)\s\(([\w\-]+)\)/) || []).to_a
+        (line.match(/\s+(.+?)\s\(([\w\-]+)\).*/) || []).to_a
```

Comment:
This regex change allows the full watch device name to be captured. It was previously coming in as just "42mm", etc.